src/core/mantle/data/gtfs.ts

```diff
@@ -0,0 +1,112 @@
+import Pbf from "pbf";
+
+import type { Data, DataRange, Feature } from "../types";
+
+import { Trips, Trip, GTFS, GTFSReader } from "./gtfsReader";
+import { f } from "./utils";
+
+let gtfs: GTFS = {};
```

Comment:
Why is this value global? I think it's better to close within local scope.

Reply:
It is but I left it like that for interval fetching where the values will be refreshed in every cycle.

Take a look at this
https://github.com/reearth/reearth-web/pull/428/commits

Reply:
I see. It looks good.